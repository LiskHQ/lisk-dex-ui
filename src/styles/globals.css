:root {
  font-size: 16px;
}

html,
body,
#__next {
  padding: 0 !important;
  margin: 0;
  height: 100%;
}

a {
  color: inherit;
  text-decoration: none;
}

* {
  box-sizing: border-box;
}

a,
.btn,
button,
span,
p,
i,
input,
select,
textarea,
li,
img,
svg path,
*::after,
*::before,
.transition-3,
h1,
h2,
h3,
h4,
h5,
h6 {
  transition: all 0.3s ease-out 0s;
}

span,
p,
a,
li.MuiMenuItem-root,
label,
label.MuiFormLabel-root,
span.MuiTypography-root,
p.MuiTypography-root,
button.MuiButton-root,
button.MuiButtonBase-root,
.MuiInput-root,
.MuiTableCell-root {
  font-family: 'Roboto', sans-serif;
}

h1,
h2,
h3,
h4,
h5,
h6,
h1.MuiTypography-root,
h2.MuiTypography-root,
h3.MuiTypography-root,
h4.MuiTypography-root,
h5.MuiTypography-root,
h6.MuiTypography-root {
  font-family: 'Roboto', sans-serif;
}