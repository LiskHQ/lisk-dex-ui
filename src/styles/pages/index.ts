import { GovernanceStyle } from "./governance/index.style";

export {
  GovernanceStyle
}