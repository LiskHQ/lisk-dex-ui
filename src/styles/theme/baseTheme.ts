export const baseTheme = {
  typography: {
    h1: {
      fontSize: "2.5rem",
      fontWeight: 600,
    },
    h2: {
      fontSize: "2rem",
      fontWeight: 600,
    },
    h3: {
      fontSize: "1.75rem",
      lineHeight: "2rem",
      fontWeight: 500,
    },
    h4: {
      fontSize: "1.25rem",
      fontWeight: 600,
    },
    h5: {
      fontSize: "1rem",
      lineHeight: "1.25rem",
      fontWeight: 500,
    },
    h6: {
      fontSize: "0.875rem",
      lineHeight: "1rem",
      fontWeight: 500,
    },
    body1: {
      fontSize: "1rem",
      lineHeight: "1.375rem",
      fontWeight: 500,
    },
    body2: {
      fontSize: "0.875rem",
      lineHeight: "1.125rem",
      fontWeight: 400,
    },
    caption: {
      fontSize: "0.875rem",
      lineHeight: "1rem",
      fontWeight: 500,
    },
    subtitle1: {
      fontSize: "1.75rem",
      lineHeight: "2rem",
      fontWeight: 500,
    },
    subtitle2: {
      fontSize: "1.25rem",
      lineHeight: "1.5rem",
      fontWeight: 400,
    }
  },
  icons: {
    sm: "1rem",
    md: "1.25rem",
    lg: "1.5rem",
  },
  breakpoints: {
    values: {
      xs: 0,
      sm: 576,
      md: 768,
      lg: 1024,
      xl: 1440,
    },
  },
  shadows: {
    sm: "0px 1px 3px rgba(0, 0, 0, 0.1)"
  }
}