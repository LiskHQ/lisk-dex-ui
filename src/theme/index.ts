export { default as theme } from "./theme";
export { default as variables } from "./variables";
