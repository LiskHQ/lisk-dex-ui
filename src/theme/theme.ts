import {
  createTheme,
  responsiveFontSizes,
  ThemeOptions,
} from "@mui/material/styles";

import variables from "./variables";

const options: ThemeOptions = {
  palette: {
    primary: variables.primary,
    secondary: variables.secondary,
  },
  typography: {
    h1: {
      fontSize: "2.5rem",
      fontWeight: 600,
    },
    h2: {
      fontSize: "2rem",
      fontWeight: 400,
    },
    h3: {
      fontSize: "1.75rem",
      fontWeight: 400,
    },
    h4: {
      fontSize: "1.25rem",
      fontWeight: 400,
    },
    h5: {
      fontSize: "1rem",
      fontWeight: 400,
    },
    h6: {
      fontSize: "0.75rem",
      fontWeight: 400,
    },
  },
  breakpoints: {
    values: {
      xs: 0,
      sm: 576,
      md: 768,
      lg: 1024,
      xl: 1440,
      xxl: 1950,
      xxxl: 2300,
    },
  },
};

const theme = responsiveFontSizes(createTheme(options));
export default theme;
