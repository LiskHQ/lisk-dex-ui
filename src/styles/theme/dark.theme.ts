import { createTheme } from '@mui/material/styles';
import { baseTheme } from './baseTheme';

const themeColors = {
  text: {
    primary: '#F2F2F2',
    secondary: '#F2F2F2',
    paragraph: '#CDCDCD',
    body: '#F2F2F2',
    heading: '#4738A6',
    button: '#FFFFFF',
    placeholder: '#A4ABB7',
  },
  lightcurve: {
    0: '#6953F4',
    10: '#5845CC',
    20: '#4738A6',
    40: '#362B7F',
    60: '#261E59',
    80: '#E6E6E6',
  },
  secondary: {
    0: '#F5981B'
  },
  primary: {
    85: '#7E6CF4',
    70: '#4F39B2',
    60: '#271B5B',
    40: '#2D2166',
    20: '#322573',
    10: '#4F39B2',
    5: '#322573',
    2.5: '#25195E',
    1: '#F4F8F9',
    0: '#FFFFFF',
  },
  opacities: {
    5: '#333333',
    10: '#3D3D3D',
    20: '#E6E6E6',
    40: '#7E7E7E',
    80: '#D4D4D4',
  },
  border: {
    primary: '#433199',
  },
  bg: {
    primary: 'linear-gradient(90deg, #140C42 0%, #211953 52.88%, #110A3A 101.53%, #140C42 101.53%)',
    secondary: '#22184C',
    helper: '#6953F4',
    proposalItem: 'linear-gradient(180deg, #2E2567 0%, rgba(37, 26, 103, 0.5) 143.01%);',
    banner: 'linear-gradient(180deg, #150556 0%, rgba(25, 9, 98, 0.9) 100%)',
    walletAddress: 'linear-gradient(180deg, #372589 0%, rgba(27, 19, 68, 0.8) 100%)',
    slider: '#322573',
  },
  success: {
    primary: '#E3F2E0',
    second: '#8DC881',
  },
  warning: {
    primary: '#F8F3D8',
    second: '#E2CA64',
  },
  info: {
    primary: '#2EA3DD',
    second: '#C0ECE8',
  },
  error: {
    primary: '#F6E4E6',
    second: '#FB8B96',
  },
} as const;

const lightTheme = createTheme({
  ...themeColors,
  ...baseTheme,
  palette: {
    mode: 'dark',
  },
})

export default lightTheme
