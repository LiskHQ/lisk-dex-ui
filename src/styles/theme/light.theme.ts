import { createTheme } from '@mui/material/styles';
import { baseTheme } from './baseTheme';

const themeColors = {
  text: {
    primary: '#282828',
    secondary: '#515661',
    paragraph: '#6B7280',
    body: '#6B7280',
    heading: '#4738A6',
    button: '#FFFFFF',
    placeholder: '#6B7280',
  },
  lightcurve: {
    0: '#6953F4',
    10: '#A196F6',
    20: '#4738A6',
    40: '#362B7F',
    60: '#261E59',
    80: '#161133',
  },
  primary: {
    85: '#7E6CF4',
    60: '#A196F6',
    40: '#E4EBF9',
    20: '#D9D8F8',
    10: '#E7E9F9',
    5: '#EEF0F8',
    2.5: '#F2F5F9',
    1: '#F4F8F9',
    0: '#6953F4'
  },
  opacities: {
    5: '#333333',
    10: '#3D3D3D',
    20: '#535353',
    40: '#7E7E7E',
    80: '#D4D4D4',
  },
  border: {
    primary: '#E6E6E6',
  },
  bg: {
    primary: '#FBFBFB',
    secondary: '#FFFFFF',
    helper: '#22184D',
    banner: 'linear-gradient(180deg, #F3F3FD 0%, rgba(238, 238, 249, 0.9) 100%)',
    walletAddress: 'linear-gradient(180deg, #EEEFF7 0%, rgba(232, 234, 245, 0.8) 100%);',
    slider: '#F2F2F2',
  },
  success: {
    primary: '#E3F2E0',
    second: '#8DC881',
  },
  warning: {
    primary: '#F8F3D8',
    second: '#E2CA64',
  },
  info: {
    primary: '#EBF7F6',
    second: '#C0ECE8',
  },
  error: {
    primary: '#F6E4E6',
    second: '#FB8B96',
  },
} as const;

const lightTheme = createTheme({
  ...themeColors,
  ...baseTheme,
  palette: {
    mode: 'light'
  },
})

export default lightTheme
