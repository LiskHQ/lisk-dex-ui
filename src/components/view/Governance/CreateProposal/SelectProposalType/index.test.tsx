import { ThemeProvider } from '@mui/material';
import { render } from '@testing-library/react';
import { SelectProposalTypeComponent } from './index';
import React from 'react';
import { lightTheme } from 'styles/theme';

function renderComponent() {
  return render(
    <ThemeProvider theme={lightTheme}>
      <SelectProposalTypeComponent />
    </ThemeProvider>
  );
}

describe('SelectProposalTypeComponent', () => {
  it('checks if the component matches the snapshot', () => {
    const { container } = renderComponent();
    expect(container).toMatchSnapshot();
  });
});

