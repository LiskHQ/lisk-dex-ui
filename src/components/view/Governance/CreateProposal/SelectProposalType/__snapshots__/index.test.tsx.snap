// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectProposalTypeComponent checks if the component matches the snapshot 1`] = `
.emotion-0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  position: relative;
  min-width: 0;
  padding: 0;
  margin: 0;
  border: 0;
  vertical-align: top;
  width: 100%;
}

.emotion-0 .MuiFormLabel-root {
  position: unset;
  color: #282828!important;
  font-size: 0.875rem;
  line-height: 1.124rem;
  font-weight: 400;
  -webkit-transform: none;
  -moz-transform: none;
  -ms-transform: none;
  transform: none;
  -webkit-transform: none;
  -ms-transform: none;
}

.emotion-0 .proposals-grid {
  margin-top: 0rem;
}

.emotion-0 .proposals-grid .proposal-box {
  height: 100%;
  padding: 1.5rem 1rem 1rem 1rem;
  border-radius: 0.375rem;
  border: 0.5px solid #E6E6E6;
  -webkit-animation-duration: 0.5s;
  animation-duration: 0.5s;
  cursor: pointer;
}

.emotion-0 .proposals-grid .proposal-box:hover {
  border: 1px solid #5845CC;
}

.emotion-0 .proposals-grid .proposal-box.selected {
  border: 1px solid #6953F4;
}

.emotion-0 .proposals-grid .proposal-box .proposal-header {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  height: 3rem;
  padding: 0 0.5rem 0 0.5rem;
}

.emotion-0 .proposals-grid .proposal-box .proposal-title {
  margin-top: 0.5rem;
  color: #282828;
}

.emotion-0 .proposals-grid .proposal-box .proposal-description {
  margin-top: 0.75rem;
  color: #6B7280;
}

.emotion-1 {
  color: rgba(0, 0, 0, 0.6);
  font-size: 1rem;
  line-height: 1.4375em;
  font-weight: 500;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  letter-spacing: 0.00938em;
  padding: 0;
  position: relative;
}

.emotion-1.Mui-focused {
  color: #1976d2;
}

.emotion-1.Mui-disabled {
  color: rgba(0, 0, 0, 0.38);
}

.emotion-1.Mui-error {
  color: #d32f2f;
}

.emotion-2 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  width: 100%;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  margin-top: -24px;
  width: calc(100% + 24px);
  margin-left: -24px;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
}

.emotion-2>.MuiGrid-item {
  padding-top: 24px;
}

.emotion-2>.MuiGrid-item {
  padding-left: 24px;
}

.emotion-3 {
  box-sizing: border-box;
  margin: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

@media (min-width:768px) {
  .emotion-3 {
    -webkit-flex-basis: 50%;
    -ms-flex-preferred-size: 50%;
    flex-basis: 50%;
    -webkit-box-flex: 0;
    -webkit-flex-grow: 0;
    -ms-flex-positive: 0;
    flex-grow: 0;
    max-width: 50%;
  }
}

@media (min-width:1024px) {
  .emotion-3 {
    -webkit-flex-basis: 50%;
    -ms-flex-preferred-size: 50%;
    flex-basis: 50%;
    -webkit-box-flex: 0;
    -webkit-flex-grow: 0;
    -ms-flex-positive: 0;
    flex-grow: 0;
    max-width: 50%;
  }
}

@media (min-width:1440px) {
  .emotion-3 {
    -webkit-flex-basis: 50%;
    -ms-flex-preferred-size: 50%;
    flex-basis: 50%;
    -webkit-box-flex: 0;
    -webkit-flex-grow: 0;
    -ms-flex-positive: 0;
    flex-grow: 0;
    max-width: 50%;
  }
}

.emotion-6 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  padding: 9px;
  border-radius: 50%;
  color: rgba(0, 0, 0, 0.6);
  color: #E6E6E6!important;
}

.emotion-6::-moz-focus-inner {
  border-style: none;
}

.emotion-6.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-6 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-6:hover {
  background-color: rgba(25, 118, 210, 0.04);
}

@media (hover: none) {
  .emotion-6:hover {
    background-color: transparent;
  }
}

.emotion-6.Mui-checked {
  color: #1976d2;
}

.emotion-6.Mui-disabled {
  color: rgba(0, 0, 0, 0.26);
}

.emotion-6.Mui-checked {
  color: #6953F4!important;
}

.emotion-7 {
  cursor: inherit;
  position: absolute;
  opacity: 0;
  width: 100%;
  height: 100%;
  top: 0;
  left: 0;
  margin: 0;
  padding: 0;
  z-index: 1;
}

.emotion-8 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-9 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1em;
  height: 1em;
  display: inline-block;
  fill: currentColor;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  -webkit-transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  font-size: 1.5rem;
  -webkit-transform: scale(1);
  -moz-transform: scale(1);
  -ms-transform: scale(1);
  transform: scale(1);
}

.emotion-10 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1em;
  height: 1em;
  display: inline-block;
  fill: currentColor;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  -webkit-transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  font-size: 1.5rem;
  left: 0;
  position: absolute;
  -webkit-transform: scale(0);
  -moz-transform: scale(0);
  -ms-transform: scale(0);
  transform: scale(0);
  -webkit-transition: -webkit-transform 150ms cubic-bezier(0.4, 0, 1, 1) 0ms;
  transition: transform 150ms cubic-bezier(0.4, 0, 1, 1) 0ms;
}

.emotion-11 {
  overflow: hidden;
  pointer-events: none;
  position: absolute;
  z-index: 0;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  border-radius: inherit;
}

.emotion-12 {
  margin: 0;
  font-size: 1rem;
  line-height: 1.25rem;
  font-weight: 500;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  letter-spacing: 0em;
}

.emotion-13 {
  margin: 0;
  font-size: 0.875rem;
  line-height: 1.125rem;
  font-weight: 400;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  letter-spacing: 0.01071em;
}

<div>
  <div
    class="MuiFormControl-root emotion-0"
  >
    <label
      class="MuiFormLabel-root MuiFormLabel-colorPrimary emotion-1"
    />
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 proposals-grid emotion-2"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-6 emotion-3"
      >
        <div
          class="proposal-box MuiBox-root emotion-4"
          data-testid="pool-incentivization-proposal-test"
        >
          <div
            class="proposal-header MuiBox-root emotion-4"
          >
            <svg
              fill="none"
              height="32"
              viewBox="0 0 32 32"
              width="32"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M32 26.1335V30.4004L28.267 32V27.7337L32 26.1335Z"
                fill="#7538D6"
              />
              <path
                d="M3.20059 17.6003H4.26698V19.7338H3.20059V17.6003Z"
                fill="#A89EF8"
              />
              <path
                d="M15.4668 17.6003H16.5339V19.7338H15.4668V17.6003Z"
                fill="#A89EF8"
              />
              <path
                d="M27.7338 17.6003H28.8001V19.7338H27.7338V17.6003Z"
                fill="#A89EF8"
              />
              <path
                d="M32 13.8672V18.1335L28.267 19.7337V15.4668L32 13.8672Z"
                fill="#7538D6"
              />
              <path
                d="M27.7338 5.33331H28.8001V7.46678H27.7338V5.33331Z"
                fill="#A89EF8"
              />
              <path
                d="M32 1.6003V5.86724L28.267 7.46682V3.20057L32 1.6003Z"
                fill="#7538D6"
              />
              <path
                d="M32 1.6003L28.267 3.20057L24.5338 1.60028L28.2669 0L32 1.6003Z"
                fill="#6953F4"
              />
              <path
                d="M28.2669 12.2669L32 13.8672L28.267 15.4668L24.5338 13.8672L28.2669 12.2669Z"
                fill="#6953F4"
              />
              <path
                d="M28.2669 24.5339L32 26.1335L28.267 27.7337L24.5338 26.1335L28.2669 24.5339Z"
                fill="#6953F4"
              />
              <path
                d="M24.5338 26.1335L28.267 27.7337V32L24.5338 30.4004V26.1335Z"
                fill="#4738A6"
              />
              <path
                d="M24.5338 13.8672L28.267 15.4668V19.7337L24.5338 18.1335V13.8672Z"
                fill="#4738A6"
              />
              <path
                d="M24.5338 1.60028L28.267 3.20057V7.46682L24.5338 5.86724V1.60028Z"
                fill="#4738A6"
              />
              <path
                d="M19.7337 30.4004L16 32V27.7337L19.7337 26.1335V30.4004Z"
                fill="#7538D6"
              />
              <path
                d="M19.7337 18.1335L16 19.7337V15.4668L19.7337 13.8672V18.1335Z"
                fill="#7538D6"
              />
              <path
                d="M15.4668 5.33331H16.5339V7.46678H15.4668V5.33331Z"
                fill="#A89EF8"
              />
              <path
                d="M19.7337 5.86724L16 7.46682V3.20057L19.7337 1.6003V5.86724Z"
                fill="#7538D6"
              />
              <path
                d="M19.7337 1.6003L16 3.20057L12.267 1.60028L16 0L19.7337 1.6003Z"
                fill="#6953F4"
              />
              <path
                d="M16 12.2669L19.7337 13.8672L16 15.4668L12.267 13.8672L16 12.2669Z"
                fill="#6953F4"
              />
              <path
                d="M16 24.5339L19.7337 26.1335L16 27.7337L12.267 26.1335L16 24.5339Z"
                fill="#6953F4"
              />
              <path
                d="M12.267 26.1335L16 27.7337V32L12.267 30.4004V26.1335Z"
                fill="#4738A6"
              />
              <path
                d="M12.267 13.8672L16 15.4668V19.7337L12.267 18.1335V13.8672Z"
                fill="#4738A6"
              />
              <path
                d="M12.267 1.60028L16 3.20057V7.46682L12.267 5.86724V1.60028Z"
                fill="#4738A6"
              />
              <path
                d="M7.46684 30.4004L3.73379 32V27.7337L7.46684 26.1335V30.4004Z"
                fill="#7538D6"
              />
              <path
                d="M7.46684 18.1335L3.73379 19.7337V15.4668L7.46684 13.8672V18.1335Z"
                fill="#7538D6"
              />
              <path
                d="M3.20059 5.33331H4.26698V7.46678H3.20059V5.33331Z"
                fill="#A89EF8"
              />
              <path
                d="M7.46684 5.86724L3.73379 7.46682V3.20057L7.46684 1.6003V5.86724Z"
                fill="#7538D6"
              />
              <path
                d="M7.46684 1.6003L3.73379 3.20057L0 1.60028L3.73374 0L7.46684 1.6003Z"
                fill="#6953F4"
              />
              <path
                d="M3.73374 12.2669L7.46684 13.8672L3.73379 15.4668L0 13.8672L3.73374 12.2669Z"
                fill="#6953F4"
              />
              <path
                d="M3.73374 24.5339L7.46684 26.1335L3.73379 27.7337L0 26.1335L3.73374 24.5339Z"
                fill="#6953F4"
              />
              <path
                d="M3.73379 27.7337V32L0 30.4004V26.1335L3.73379 27.7337Z"
                fill="#4738A6"
              />
              <path
                d="M3.73379 15.4668V19.7337L0 18.1335V13.8672L3.73379 15.4668Z"
                fill="#4738A6"
              />
              <path
                d="M3.73379 3.20057V7.46682L0 5.86724V1.60028L3.73379 3.20057Z"
                fill="#4738A6"
              />
              <path
                d="M3.20059 7.46678H4.26698V13.8672H3.20059V7.46678Z"
                fill="#A89EF8"
              />
              <path
                d="M15.4668 7.46678H16.5339V13.8672H15.4668V7.46678Z"
                fill="#A89EF8"
              />
              <path
                d="M27.7338 7.46678H28.8001V13.8672H27.7338V7.46678Z"
                fill="#A89EF8"
              />
              <path
                d="M3.20059 19.7338H4.26698V26.1334H3.20059V19.7338Z"
                fill="#A89EF8"
              />
              <path
                d="M15.4668 19.7338H16.5339V26.1334H15.4668V19.7338Z"
                fill="#A89EF8"
              />
              <path
                d="M27.7338 19.7338H28.8001V26.1334H27.7338V19.7338Z"
                fill="#A89EF8"
              />
              <path
                d="M5.86721 16H13.8672V17.0671H5.86721V16Z"
                fill="#A89EF8"
              />
              <path
                d="M5.86721 3.73372H13.8672V4.80011H5.86721V3.73372Z"
                fill="#A89EF8"
              />
              <path
                d="M5.86721 28.2669H13.8672V29.3333H5.86721V28.2669Z"
                fill="#A89EF8"
              />
              <path
                d="M18.1334 16H26.1334V17.0671H18.1334V16Z"
                fill="#A89EF8"
              />
              <path
                d="M18.1334 3.73372H26.1334V4.80011H18.1334V3.73372Z"
                fill="#A89EF8"
              />
              <path
                d="M18.1334 28.2669H26.1334V29.3333H18.1334V28.2669Z"
                fill="#A89EF8"
              />
            </svg>
            <span
              class="MuiRadio-root MuiRadio-colorPrimary MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root emotion-6"
            >
              <input
                class="PrivateSwitchBase-input emotion-7"
                name="radio-buttons"
                type="radio"
                value="POOL_INCENTIviZATION"
              />
              <span
                class="emotion-8"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium emotion-9"
                  data-testid="RadioButtonUncheckedIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium emotion-10"
                  data-testid="RadioButtonCheckedIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root emotion-11"
              />
            </span>
          </div>
          <h5
            class="MuiTypography-root MuiTypography-h5 proposal-title emotion-12"
          >
            Pool incentivization proposal
          </h5>
          <p
            class="MuiTypography-root MuiTypography-body2 proposal-description emotion-13"
          >
            A pool incentivization proposal is a proposal type in the DEX Governance module that requests modifying the list of incentivized liquidity pools. Once accepted, this proposal has immediate on-chain consequences.
          </p>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-6 emotion-3"
      >
        <div
          class="proposal-box MuiBox-root emotion-4"
          data-testid="universal-proposal-test"
        >
          <div
            class="proposal-header MuiBox-root emotion-4"
          >
            <svg
              fill="none"
              height="32"
              viewBox="0 0 32 32"
              width="32"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M17.0456 0.0334464C7.40093 -0.582814 -0.582814 7.40093 0.0334464 17.0456C0.542914 25.0174 6.98256 31.4565 14.9544 31.9666C24.5984 32.5828 32.5828 24.5991 31.9666 14.9544C31.4565 6.98255 25.0174 0.542914 17.0456 0.0334464ZM23.244 23.9388C23.8735 21.7766 24.2492 19.2902 24.2994 16.6153H30.7535C30.6115 20.0446 29.2955 23.1749 27.196 25.6155C26.233 24.9408 24.8868 24.3698 23.244 23.9388ZM4.80334 25.6155C2.70454 23.1749 1.38784 20.0446 1.24649 16.6153H7.69996C7.75021 19.2902 8.12587 21.7766 8.75533 23.9388C7.11196 24.3704 5.76574 24.9408 4.80334 25.6155ZM8.75218 8.07185C8.12462 10.231 7.75021 12.7142 7.69996 15.3841H1.24649C1.38721 11.9711 2.69197 8.85458 4.77319 6.41906C5.75255 7.08244 7.1082 7.6453 8.75218 8.07185ZM15.3841 15.3841H8.93059C8.98336 12.8373 9.36216 10.4339 9.98094 8.35454C11.6444 8.68999 13.4888 8.88096 15.3841 8.91552V15.3841ZM15.3841 16.6153V23.0845C13.485 23.119 11.6406 23.3131 9.98345 23.6524C9.36342 21.5712 8.98399 19.1652 8.93059 16.6153H15.3841ZM16.6153 16.6153H23.0688C23.016 19.1652 22.636 21.5712 22.0166 23.6524C20.3581 23.3131 18.5144 23.119 16.6153 23.0845V16.6153ZM16.6153 15.3841V8.91489C18.5106 8.88096 20.355 8.68936 22.0184 8.35391C22.6372 10.4339 23.016 12.8373 23.0688 15.3841H16.6153ZM16.6153 7.68425V1.29298C18.6362 1.67681 20.4448 3.92135 21.6302 7.17855C20.0842 7.48009 18.3736 7.65158 16.6153 7.68425ZM15.3841 1.29298V7.68425C13.6257 7.65158 11.9158 7.48009 10.3692 7.17855C11.5546 3.92135 13.3631 1.67618 15.3841 1.29298ZM15.3841 24.3151V30.707C13.3644 30.3238 11.5571 28.0818 10.3717 24.8284C11.9126 24.5224 13.6226 24.3484 15.3841 24.3151ZM16.6153 30.7064V24.3151C18.3768 24.3484 20.0861 24.5224 21.6277 24.8284C20.4423 28.0812 18.635 30.3238 16.6153 30.7064ZM24.2994 15.3841C24.2492 12.7142 23.8748 10.231 23.2478 8.07122C24.8912 7.64467 26.2468 7.08244 27.2262 6.41906C29.3074 8.85458 30.6122 11.9711 30.7529 15.3841H24.2994ZM26.375 5.50127C25.5137 6.04717 24.3082 6.52837 22.8715 6.89712C22.0995 4.71351 21.051 2.93697 19.816 1.73335C22.3275 2.40552 24.5777 3.72473 26.375 5.50127ZM9.12785 6.89712C7.69179 6.529 6.48628 6.04717 5.62439 5.50127C7.42229 3.72473 9.67186 2.40614 12.184 1.73335C10.949 2.93697 9.8999 4.71414 9.12785 6.89712ZM5.6558 26.5283C6.49884 25.9742 7.69493 25.4861 9.13162 25.1129C9.90367 27.2915 10.9509 29.0643 12.184 30.266C9.68631 29.5976 7.44805 28.2891 5.6558 26.5283ZM22.8678 25.1129C24.3044 25.4861 25.5005 25.9742 26.3442 26.5283C24.5513 28.2897 22.3137 29.5976 19.816 30.266C21.0491 29.0643 22.0957 27.2915 22.8678 25.1129Z"
                fill="url(#paint0_linear_4113_2378)"
              />
              <defs>
                <lineargradient
                  gradientUnits="userSpaceOnUse"
                  id="paint0_linear_4113_2378"
                  x1="16"
                  x2="16"
                  y1="0"
                  y2="32"
                >
                  <stop
                    stop-color="#7968E7"
                  />
                  <stop
                    offset="1"
                    stop-color="#45369F"
                  />
                </lineargradient>
              </defs>
            </svg>
            <span
              class="MuiRadio-root MuiRadio-colorPrimary MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root emotion-6"
            >
              <input
                class="PrivateSwitchBase-input emotion-7"
                name="radio-buttons"
                type="radio"
                value="UNIVERSAL"
              />
              <span
                class="emotion-8"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium emotion-9"
                  data-testid="RadioButtonUncheckedIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium emotion-10"
                  data-testid="RadioButtonCheckedIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root emotion-11"
              />
            </span>
          </div>
          <h5
            class="MuiTypography-root MuiTypography-h5 proposal-title emotion-12"
          >
            Universal proposal
          </h5>
          <p
            class="MuiTypography-root MuiTypography-body2 proposal-description emotion-13"
          >
            A universal proposal is a proposal type in the DEX Governance module that can request arbitrary DEX-related actions, for example, changes in protocol, UI or even project strategy. Once accepted, this proposal is acted upon off-chain.
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
